### PR TITLE
Implement support for Squiggly annotations

### PR DESCRIPTION
**Pull request by timvandermeij**
Notice that the reduced test case provided with this patch now renders just like Adobe Reader/Acrobat and without any warnings in the console.

### CODE REVIEW

File: web/annotation_layer_builder.css [hunk 1]
```diff
@@ -73,6 +73,10 @@
   cursor: pointer;
 }
 
+.annotationLayer .squigglyAnnotation {
```
Comment (Snuffleupagus) [inline web/annotation_layer_builder.css]: Given that the rules are identical, can't you just do this instead:
```css
.annotationLayer .underlineAnnotation,
.annotationLayer .squigglyAnnotation,
.annotationLayer .strikeoutAnnotation {
  cursor: pointer;
}
```

**Comment by Snuffleupagus** [comment]
r=me, with less duplication in the CSS.

File: web/annotation_layer_builder.css (hunk 1 above)
Comment (timvandermeij) [inline web/annotation_layer_builder.css]: Good point. Done in the new commit.

**Comment by timvandermeij** [comment]
Landing with r=Snuffleupagus and r=me.

/botio makeref

**Pull request merged**